spec/acceptance/docker_spec.rb

```diff
@@ -43,12 +43,25 @@
   context 'When adding system user', win_broken: broken do
     let(:pp) do
       "
-             class { 'docker': #{docker_arg}
+             class { 'docker': #{docker_args}
                docker_users => ['user1']
              }
      "
     end
 
+    context 'Checking root_dir value' do
+      let(:pp) do
+        "class { 'docker': #{docker_args}}"
+      end
+
+      it 'is good' do
+        apply_manifest(pp, catch_failures: true)
+        run_shell('cat C:/ProgramData/docker/config/daemon.json') do |r|
```

Comment:
There's helpers that can verify the content of the file in `puppet-rspec` - see [here](https://github.com/puppetlabs/puppetlabs-apache/blob/f8af65497f7198eac5e6eedd4bae529f561f2da4/spec/acceptance/mod_ldap_spec.rb#L15-L17)

...although this is fine 👍 

One thing I might suggest is verifying the value of `data-root` too?